docs/tutorials/kubernetes/kyverno.md

```diff
@@ -59,36 +70,10 @@ kubectl apply -f vuln-attestation.yaml
 
 To ensure that the policy worked, we can deploye an example deployment file with our container image:
```

Comment:
Although not in the scope of this PR, there is a typo and it would be good to fix this as well.
```suggestion
To ensure that the policy worked, we can deploy an example deployment file with our container image:
```

Reply:
Oh gosh, thank you for reading over it, will fix the spelling mistakes